packages/browser/services/mongodb-remote/src/RemoteMongoCollection.ts

```diff
@@ -245,15 +244,12 @@ export default interface RemoteMongoCollection<DocumentT> {
    * [EventSource Browser Compatibility on MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventSource#Browser_compatibility)
    * for information on which browsers support EventSource.
    *
```

Comment:
@cbush when this is merged, it would nice if you could clean this up and add code snippets